modules/aws_flows/variables.tf

```
@@ -0,0 +1,17 @@
+variable "flow_logs_bucket_arns" {
```

Keep the naming of variables with the naming of attributes of the provider resource:
https://github.com/illumio/terraform-provider-illumio-cloudsecure/blob/010d3d1dc7ac76005ab893aa6d4064968f4ae533/api/schema/aws_flow_logs_s3_bucket.go#L32
```suggestion
variable "s3_bucket_arns" {
```